package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "material-ui-rating",
-  "version": "3.1.0",
+  "version": "3.1.1",
```

Comment:
Please let me bump the version, I'm using `np` for that.

Reply:
this change came from your earlier commit. I'm sitll not sure how or why it got mixed to my PR.

and yes, I'm doing the PR to master. should I choose another branch?

![image](https://user-images.githubusercontent.com/6963212/59561223-58885f00-9026-11e9-9420-9b185f747d67.png)

Reply:
@goldylucks Doesn't really matter what the branch is on your side 😉 I hope you don't mind me comitting on your master branch (which is why PRs aren't done from the master branch usually; the maintainers of the upstream repo will get commit access via this PR).